templates/superset_python_path/superset_config.py.j2

```diff
@@ -138,10 +132,9 @@ PATCHUP_EMAIL_BASE = {{ superset_patchup_email_base }}
 ALLOWED_EXTENSIONS = set(['{{ superset_allowed_extensions|join("', '") }}'])
 {% endif %}
 
-{% if superset_flask_app_mutator%}
-# Allow adding custom views from superset_patchup
-def import_custom_view(myapp):
-    from superset_patchup import views
+{% if superset_use_ketchup %}
+import sys
+import superset_patchup
 
-FLASK_APP_MUTATOR = import_custom_view
+superset_patchup.add_ketchup(sys.modules[__name__])
```

Comment:
@moshthepitt Can you take another look? - this patch requires all the hookup logic get shoved into the -patchup module itself.  See: https://github.com/onaio/superset-patchup/pull/35/commits/bb355344f21cc76d6fff59c2629715881b9cc3e8